Dockerfile

```diff
@@ -29,7 +29,7 @@ ADD . /build/
 RUN echo "Starting the script sections" && \
 		dart --version && \
 		pub get && \
-        pub run dependency_validator -i coverage && \
+				pub run dependency_validator -i coverage,build_runner,build_test,build_web_compilers && \
```

Comment:
#nit this has 4 tabs and the previous lines have 2

Reply:
🤦‍♀️